commands/pull_request.go

```diff
@@ -208,7 +212,13 @@ func pullRequest(cmd *Command, args *Args) {
 		}
 
 		utils.Check(err)
+
 		pullRequestURL = pr.HTMLURL
+
+		if flagPullRequestAssignee != "" {
+			err = client.UpdateIssue(baseProject, pr.Number, flagPullRequestAssignee)
```

Comment:
Since this call basically just sets an assignee, the method name should reflect that; e.g. `UpdateIssueAssignee()`